docs/api/target_properties.rst

```diff
@@ -190,6 +190,44 @@ Output is of the form for each property:
     blt_print_target_properties(TARGET foo CHILDREN TRUE PROPERTY_NAME_REGEX "CXX")
     blt_print_target_properties(TARGET foo PROPERTY_NAME_REGEX "CXX" PROPERTY_VALUE_REGEX "ON")
 
+.. _blt_print_variables:
+
+blt_print_variables
+~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: cmake
+
+    blt_print_variables(NAME_REGEX  <regular_expression_string>
+                        VALUE_REGEX <regular_expression_string>
+                        [IGNORE_CASE])
+
+Prints variables and their values in the current scope, with optional 
+regex filtering on the variable names and values.
+
+NAME_REGEX
+  Regular expression to apply to variable names (all by default)
+
+VALUE_REGEX
+  Regular expression to apply to variable values (all by default)
+
+IGNORE_CASE
+  Optionally ignore case of variable names and values when filtering
+
+
+Output is of the following form for each variable:
```

Comment:
This is weirdly getting bolded in the documentation.

Reply:
hmm.. its also like this on the 'blt_print_target_properties' macro..

Reply:
I'll play around with it. 
Could also be that the colon at the end of the line has special semantics for sphinx (?)